src/components/modal/modal.module.scss

```diff
@@ -1,27 +1,11 @@
 @import '~theme/base/variables';
 @import '~theme/base/mediaqueries';
 
-.modal {
-    position: fixed;
-    top: 0;
-    left: 0;
-    width:100%;
-    height: 100%;
-    background: rgba($color-black, .3);
-    display: flex;
-    align-items: center;
-    justify-content: center;
-}
+.bgColor { background: rgba($color-black, .3); }
```

Comment:
We might want to create a plugin for this, to allow using this same styles somewhere else.

Reply:
Yes, I agree!

- Does it mean the rest of the PR is okay? *if that's the case,*
  - Does this line that you commented should block the PR?
  - Should we merge and create tech debt?

Reply:
Yeah... I'm thinking in the plugin we should move all the styles related to the modal, and use the tailwind utilities there to set the responsive styles.

Reply:
I'll be waiting for news then! Okay? @crysfel